.circleci/config.yml

```diff
@@ -0,0 +1,133 @@
+            password: $DOCKERHUB_PASSWORD
+commands:
+  install_python2:
+    steps:
+      - run:
+          name: Install Python 2
+          command: |
+            sudo apt-get update
+            sudo apt-get install python2-minimal
+            sudo update-alternatives --install /usr/bin/python python /usr/bin/python2 10
+  test:
+    steps:
+      - run:
+          name: Test
+          # The "cimg/node:*-browsers" images run Xvfb on :99 at startup, but that doesn't seem to be documented.
+          # If this starts failing, that might no longer be true.
+          # In that case, try: xvfb-run -e /dev/stdout npm test
+          command: DISPLAY=:99 npm test
+  deploy_gh_pages:
+    steps:
+      - when:
+          condition:
+            not:
+              matches: { pattern: "^renovate/.*", value: <<pipeline.git.branch>> }
+          steps:
+            - run:
+                name: Deploy to GH Pages
+                command: |
+                  git config --global user.email $(git log --pretty=format:"%ae" -n1)
+                  git config --global user.name $(git log --pretty=format:"%an" -n1)
+                  npm run deploy -- \
+                    -r https://${GH_TOKEN}@github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}.git
+  deploy_npm:
+    steps:
+      - when:
+          condition:
+            or:
+              - equal: [ master, <<pipeline.git.branch>> ]
+              - equal: [ develop, <<pipeline.git.branch>> ]
+              - matches: { pattern: "^hotfix.*", value: <<pipeline.git.branch>> }
+              - matches: { pattern: "^release.*", value: <<pipeline.git.branch>> }
+          steps:
+            - run:
+                # This relies on deploy_gh_pages to set git user info first
+                name: Deploy to NPM
+                command: |
+                  export RELEASE_VERSION="0.1.0-prerelease.$(date +'%Y%m%d%H%M%S')"
+                  if [[ "$CIRCLE_BRANCH" == hotfix/* ]]; then
+                    export NPM_TAG=hotfix
+                  else
+                    export NPM_TAG=latest
+                  fi
+                  echo "Deploying version $RELEASE_VERSION to $NPM_TAG"
+                  npm --no-git-tag-version version $RELEASE_VERSION
+                  npm set //registry.npmjs.org/:_authToken=$NPM_TOKEN
+                  npm publish --tag $NPM_TAG
+                  if npm info | grep -q $RELEASE_VERSION; then
+                    git tag $RELEASE_VERSION
+                    git push \
+                      https://${GH_TOKEN}@github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}.git \
+                      $RELEASE_VERSION
+                  fi
+
+jobs:
+  default:
+    <<: *defaults
+    environment:
+      # DETECT_CHROMEDRIVER_VERSION only works if Chrome is installed before installing NPM packages
+      DETECT_CHROMEDRIVER_VERSION: "true"
+    steps:
+      - checkout
+      - install_python2
+      - browser-tools/install-chrome
```

Comment:
Do we want to specify a node_version here to match 16.18 ? 

ie. 

```
- node/install:
          install-yarn: true
          node-version: '16.13'
   ```

Reply:
The `node/install-packages` command only installs dependencies, like `node_modules` stuff. It'll just use the Node version that already exists in the image (the `cimg/node:16.18-browsers` from `defaults` above, like you said) so everything should stay consistent.

The `node/install` command could introduce inconsistency like you're suggesting, especially if we update one version number (like in `defaults`) and forget to update the other (here), so I'd prefer to avoid that